src/sentry/receivers/outbox/control.py

```diff
@@ -113,3 +114,13 @@ def process_mark_invalid_sso(object_identifier: int, shard_identifier: int, **kw
     other_member.flags.sso__invalid = True
     other_member.flags.sso__linked = False
     organization_service.update_membership_flags(organization_member=other_member)
+
+
+@receiver(process_control_outbox, sender=OutboxCategory.ISSUE_COMMENT_UPDATE)
+def process_issue_email_reply(shard_identifier: int, payload: Any, **kwds):
+    issue_service.upsert_issue_email_reply(
```

Comment:
@RyanSkonnord Definitely another good example of where our new absstraction would help!